src/utils/cplay.c

```diff
@@ -75,7 +76,15 @@
 #include "tinycompress/tinymp3.h"
 #include "tinycompress/tinywave.h"
 
-static int verbose;
+enum {
+	DO_NOTHING = -1,
+	DO_PAUSE,
+	DO_RESUME
```

Comment:
DO_PAUSE_PUSH, DO_PAUSE_RELEASE

using resume leads to confusions with suspend/resume.

Reply:
Fixed